zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/routing/DbRoutingState.java

```diff
@@ -47,7 +47,11 @@ public DbRoutingState(
   @Override
   public Set<Integer> currentPartitions() {
     key.wrapString(CURRENT_KEY);
-    return columnFamily.get(key).getPartitions();
+    final var currentRoutingInfo = columnFamily.get(key);
+    if (currentRoutingInfo == null) {
```

Comment:
:question:  When would this be null? Isn't it always initialized with the static partition count?